### DISCUSSION TITLE
在哪里可以看到完整的入库清单吗

### DISCUSSION THREAD

**Discussion by xkbkx5904** [pjy612/SteamManifestCache#875] [Question / Answer]
新建share帖之前只是搜索了一下appid，不确定是否入过库了。因为只能看见[最新6000条](https://github.com/pjy612/SteamManifestCache/discussions/798)

**Comment by pjy612** [comment]
* 没在 free 列表里，入库时 提示 缺密钥 或 无配置
* 项目中 搜 appid 搜不到
满足 条件的 基本都可以分享
